benchmarks/setup/cloud-default/Makefile

```diff
@@ -1,45 +1,47 @@
+namespace ?= default
```

Comment:
❓ Why a safe assignment and not a regular assignment (i.e. `=`)? I'm not that familiar with the assignment syntax of Makefiles, and I couldn't find a clear answer on why you'd want to do this. As I understand it, you'll replace the `default` literal anyways with `sed`.

Reply:
One reason is that I just copied it from the other Makefile https://github.com/camunda-cloud/zeebe/blob/develop/benchmarks/setup/default/Makefile#L1

The other reason is I think to set the variable also from outside, like https://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make

Reply:
Thanks 👍  I learned something today 🎉